src/interfaces/dab_registries/canister_registry.ts

```diff
@@ -0,0 +1,42 @@
+import type { Principal } from '@dfinity/principal';
+import RegistryStandard from './registry_standard';
+export type CanisterCategory = { 'NFT' : null } |
```

Comment:
CanisterCategory is no longer an enum. It's just string and it can be anything.

Reply:
if this is for the older version we should keep it.

Reply:
not really needed anymore